file-formats/dtdc/type/zif_aff_dtdc_v1.intf.abap

```diff
@@ -0,0 +1,16 @@
+INTERFACE zif_aff_dtdc_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Dynamic Cache</p>
+    "! Dynamic cache
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60,
```

Comment:
Just to be sure. Is key user tooling supported fro dynamic caches?

Reply:
> Just to be sure. Is key user tooling supported for dynamic caches?

At least DTDC is enabled for the API release API (configuration in table ars_lang_objtype).